Add V2ex.MAUI project to Samples

rwecho opened the pull request:
I created an MAUI application and hope to add the Samples. 
The V2ex.MAUI is made by MAUI completely, with MVVM, Abp Framework, communityToolkit/Maui. 

Thanks.

jsuarezruiz reviewed (commented):
Could you review the changes?. There are changes in many lines, I think the majority due to spacing but it is difficult to see the important change.

Thanks!

rwecho commented:
@jsuarezruiz  I'm sorry, The file was auto-formatted by vscode. 
I would split the commit more clearly

rwecho commented:
@jsuarezruiz  I removed the auto-formatted code and just added the project instruction.

If the readme needs to format, we can add a new PR to resolve it.
